internal/fs/buffer/inmemory_write_buffer_test.go

```diff
@@ -15,11 +15,26 @@
 package buffer
 
 import (
+	"bytes"
+	"math/rand"
+	"strings"
 	"testing"
+	"time"
 
 	. "github.com/jacobsa/ogletest"
 )
 
+const (
+	KiB                     = 1024
+	smallContent1           = "Taco"
```

Comment:
smallContent{1-3} are always cast to `byte[]` everywhere ? It's a good idea to just create them with that type right here.

Reply:
go doesn't allow creation of byte[] const

Reply:
Okay, not a const may be, but a var can be created as [here](https://stackoverflow.com/questions/13137463/declare-a-constant-array#:~:text=There%20is%20no%20such%20thing,complex%20constants%2C%20and%20string%20constants.)